ssntp/ssntp.go

```diff
@@ -1151,14 +1152,41 @@ func (config *Config) port() uint32 {
 	return port
 }
 
+func discoverCertificate(certPrefix string) (string, error) {
+	ciaoCertsPath := "/etc/pki/ciao/"
```

Comment:
@obedmr Could you please make that one a ssntp const, along with the rest of the default certificate names:
`const defaultCertPath = "/etc/pki/ciao/"`